I commited on main by mistake

DragSama opened the pull request.

Loli-Killer commented on tg_bot/modules/global_bans.py:
```diff
@@ -128,7 +132,7 @@ def gban(bot: Bot, update: Update, args: List[str]):
     log_message = "#GBANNED" \
                   "\n<b>Originated from:</b> {}" \
-                  "\n<b>Admin:</b> {}" \
+                  "\n<b>Bot-Admin:</b> {}" \
```
I think it's fine as it. Don't think it needs any changes

nitin221 replied:
Ok 🙂

Loli-Killer commented on tg_bot/modules/global_bans.py:
```diff
@@ -153,7 +157,7 @@ def gban(bot: Bot, update: Update, args: List[str]):
 
     else:
-        send_to_list(bot, SUDO_USERS + SUPPORT_USERS, log_message, html=True)
+        send_to_list(bot, SUDO_USERS + SUPPORT_USERS + DEV_USERS, log_message, html=True)
```
Don't need to add DEV_USERS to list since SUDO_USERS already contain dev user list as well.
See [this](https://github.com/AnimeKaizoku/SaitamaRobot/blob/master/tg_bot/__init__.py#L130).

nitin221 replied:
Lmao i missed that point

Loli-Killer commented on tg_bot/modules/global_bans.py:
```diff
@@ -200,7 +204,7 @@ def gban(bot: Bot, update: Update, args: List[str]):
 
     try:
-        bot.send_message(user_id, "You have been globally banned from all groups where I have administrative permissions. If you think that this was a mistake, you may appeal your ban here: @OnePunchSupport", parse_mode=ParseMode.HTML)
+        bot.send_message(user_id, "You have been globally banned from all groups where I'm appoint as Admin. If you think that this was a mistake, you may appeal for Ungban requet here: @OnePunchSupport", parse_mode=ParseMode.HTML)
```
Again, don't think this need changes as well.
Not admin always = can ban.
You would need the ban permission as well to be able to ban so the word "administravtive permissions" is ok.

nitin221 replied:
Kk lol just  specific points urself no need of anything and close the poll request

TsunayoshiSawada replied:
Although we can rewrite that string a little bit in the future, I was thinking of writing that line again entirely.

Loli-Killer commented on runtime.txt:
```diff
@@ -1 +1 @@
-python-3.6.4
+python-3.6.10
```
Any reason for this?
I think it works fine with 3.6.4

nitin221 replied:
Naah ! Not any big reason just a security update if anyone deploying on heroku

Loli-Killer commented:
Either change the points I mentioned and I'll merge it later. (If you want to have your name in the commit history is what I guess you want)
Or I can just edit those specific points myself and close the pull request. (Your name won't be in the commit history)

nitin221 commented:
U was right 🙂

TsunayoshiSawada commented on tg_bot/modules/global_bans.py:
```diff
@@ -244,9 +248,9 @@ def ungban(bot: Bot, update: Update, args: List[str]):
-                  "\n<b>Unbanned User ID:</b> {}" \
+                  "\n<b>Bot-Admin:</b> {}" \
+                  "\n<b>Ungbanned User:</b> {}" \
+                  "\n<b>Ungbanned User ID:</b> {}" \
```
These are actually raw commits. 
When loli or any of our devs add stuff, they dont finalize the translation right away, they just add placeholder text. 

We then test this our and edit the translation strings over time.

nitin221 replied:
Okkk

TsunayoshiSawada committed:
I made booboos cause see reasons beluwu (#32)

1. I is noob  
2. Typed a word in an open vs code py on wrong monitor and didnt notice 
It is requested you dont go through this commit due to uh..reasons. >.>

TsunayoshiSawada committed:
Merge pull request #38 from AnimeKaizoku/shiken

Pulling quick updates

TsunayoshiSawada committed:
Merge pull request #39 from AnimeKaizoku/shiken

Added help documentation for cleanservice messages

DragSama committed:
minor correction to setup

TsunayoshiSawada committed:
Merge pull request #42 from AnimeKaizoku/shiken

Pull updates to master

DragSama committed:
Allowed Admins to use disabled commands

DragSama committed:
Merge pull request #43 from AnimeKaizoku/shiken

> @DragSama Oof size 100
delete REEEEE

DragSama committed:
Merge pull request #44 from AnimeKaizoku/shiken

[Critical] Infinite running regex check

DragSama committed:
Merge pull request #45 from AnimeKaizoku/shiken

[Critical] more checks for regex

DragSama committed:
Merge pull request #46 from AnimeKaizoku/shiken

added missing .

DragSama committed:
fixed fpromote
